docs/contributors.rst

```diff
@@ -0,0 +1,41 @@
+ | # # # Distribution Statement A. Approved for public release. Distribution unlimited.
+ | # # #
+ | # # # Author:
+ | # # # Naval Research Laboratory, Marine Meteorology Division
+ | # # #
+ | # # # This program is free software: you can redistribute it and/or modify it under
+ | # # # the terms of the NRLMMD License included with this program. This program is
+ | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+ | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+ | # # # for more details. If you did not receive the license, for more information see:
+ | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+GeoIPS Contributors Guide
+=========================
+
+The GeoIPS Team encourages active participation by the user and developer community,
+so we welcome any and all feedback on and updates to the code base.
+
+Each step in this process can be completed by one or more individuals (we encourage collaboration!),
+so feel free to submit Issues even if you have no intention of resolving it yourself. And feel free to
```

Comment:
Suggested grammar change: "Each step in this process can be completed by one or more individuals (we encourage collaboration!) Feel free to submit an issue if you have no intention of resolving it yourself."

Reply:
Done! Feel free to resolve these conversations if you're good with the changes.